packages/now-client/src/types.ts

```diff
@@ -95,13 +95,10 @@ export interface DeploymentGithubData {
   autoJobCancelation: boolean;
 }
 
-interface LegacyNowConfig {
-  type?: string;
-  aliases?: string | string[];
-}
-
-export interface NowConfig extends LegacyNowConfig {
+export interface NowConfig {
+  [key: string]: any;
```

Comment:
Is there a reason why `[key: string]: any;` was added here? Can we add any additional properties instead?

Reply:
Had it temporarily to fix something, but removed it now 👍 
and even more